src/app/store/Category/Category.dispatcher.js

```diff
@@ -27,10 +27,12 @@ class CategoryDispatcher extends RequestDispatcher {
                 filters
             }
         } = data;
+        const { categoryUrlPath } = options;
 
         if (category) { // If category details are updated, reset all data
```

Comment:
Please double check if CategoryList is not being requested every time. Just make sure.

Reply:
Double checked. Not requesting.